app/main/views/briefs.py

```diff
@@ -356,19 +356,34 @@ def view_response_result(brief_id):
         supplier_id=current_user.supplier_id
     )['briefResponses']
 
+    legacy_brief = True
```

Comment:
I was hoping to we wouldn't need to use the feature flag for this view but you are right that we do need to because we may have no brief response so we can't look at `essentialRequirementsMet`. Shame